packages/perseus/src/widgets/interactive-graphs/axis-ticks.tsx

```diff
@@ -41,6 +56,21 @@ const XGridTick = ({x}: {x: number}) => {
                 y2={yPosition - tickSize / 2}
                 style={tickStyle}
             />
+            {
+                // TODO (LEMS-1891): Negative one is a special case as the labels can
+                // overlap with the axis line. We should handle this case more gracefully.
+            }
+            {x !== -1 && (
+                <text
+                    height={20}
+                    width={50}
+                    textAnchor="middle"
```

Comment:
We want middle here, as we want to ensure the x axis tick labels are centered under their appropriate 'tick'.